### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @tadatuta @tormozz48

### CODE REVIEW

File: server/forum.js [hunk 1]
```diff
@@ -114,6 +114,22 @@ module.exports = function(pattern, options) {
                     req.__data = req.__data || {};
                     req.__data.forum = values;
 
+                    // i18 object for page title
+                    var i18n = {
+                            ru: {
+                                title: 'Форум / БЭМ. Блок, Элемент, Модификатор'
+                            },
+                            en: {
+                                title: 'BEM. Block, Element, Modifier'
```
Comment (tadatuta) [inline server/forum.js]: why no `Forum` for Eng?

File: server/forum.js [hunk 2]
```diff
@@ -114,6 +114,22 @@ module.exports = function(pattern, options) {
...
+                            }
+                        },
+                        lang = req.headers['accept-language'].substr(0,2),
```
Comment (tadatuta) [inline server/forum.js]: do we have 100% that `accept-language` is always there?

**Comment by tavriaforever** [comment]
Update

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**